src/quizzes/chapter-3/3.3/nexus-maven-quiz.js

```diff
@@ -0,0 +1,28 @@
+const rawQuizdown = `
+---
+shuffleQuestions: true
+shuffleAnswers: true
+---
+
+# Order the following components into a pipeline that correctly utilizes the tools you have been working with in this section:
+
+1. Checkout code from repo
+2. Build artifact using ____ (place answer in the next slot) 
+3. Maven
+3. Run automated testing
+4. Deploy to ____ (place answer in the next slot)
+5. Nexus
```

Comment:
Really just a comment but making "order the following" questions is so easy!

Reply:
It really is! It's my favorite question type that Quizdown offers but it's the hardest to implement into a good question in my opinion.